Makefile

```diff
@@ -57,6 +57,7 @@ ifeq ($(HAVE_SFC),1)
 DRIVER_SUBDIRS += src/driver/linux_net/drivers/net/ethernet/sfc
 endif
 
+ALL_SUBDIRS := $(DRIVER_SUBDIRS) $(LIB_SUBDIRS)
```

Comment:
I'd like for this makefile to (eventually) be used to launch the userspace build too, so can we please have a name that is less likely to be confusing in that future?